frontend/env-local.js

```diff
@@ -0,0 +1 @@
+export const URL = "127.0.0.1";
```

Comment:
Do you want to check in this file? Nothing sensitive in here now, just wondering